### PR TITLE
[RFR] Dismiss notifications

### PR DESCRIPTION
**Pull request by psav**
Dismiss notifications because they cause us issues

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3e66b73b0693de369169e10578f287886d60ee19:

`cfme/utils/appliance/implementations/ui.py`:
- :heavy_exclamation_mark: Line 68:1: **W293** *blank line contains whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/utils/appliance/implementations/ui.py [hunk 1]
```diff
@@ -54,6 +54,18 @@ def get_rails_error(self):
...
 class MiqBrowserPlugin(DefaultPlugin):
     ENSURE_PAGE_SAFE = jsmin('''\
+        try {
+            var eventNotificationsService = angular.element('#notification-app')
+                .injector().get('eventNotifications');
+            eventNotificationsService.clearAll(
+                ManageIQ.angular.eventNotificationsData.state.groups[0]
+            );
+            eventNotificationsService.clearAll(
+                ManageIQ.angular.eventNotificationsData.state.groups[1]
```
Comment (mfalesni) [inline cfme/utils/appliance/implementations/ui.py]: So just to clarify, why 0 and 1? Can't we iterate over all? Any speciifc reason?

Comment (psav) [reply cfme/utils/appliance/implementations/ui.py]: events and tasks
I just felt iterating added another level of complexity we didn't need at this stage

File: cfme/utils/appliance/implementations/ui.py [hunk 2]
```diff
@@ -54,6 +54,18 @@ def get_rails_error(self):
 
 class MiqBrowserPlugin(DefaultPlugin):
     ENSURE_PAGE_SAFE = jsmin('''\
+        try {
```
Comment (jamesooden) [inline cfme/utils/appliance/implementations/ui.py]: Could we leave a little comment here explaining how problematic these notifications are?

File: cfme/utils/appliance/implementations/ui.py (hunk 1 above)
Comment (mfalesni) [reply cfme/utils/appliance/implementations/ui.py]: sure, just maybe add a comment that clarifies this.

**Pull request merged**